SayTheirNames/Source/Controller/Home/Person/Views/PersonNewsTableViewCell.swift

```diff
@@ -122,13 +124,19 @@ class PersonNewsTableViewCell: UITableViewCell {
         cellType = type
         collectionView.delegate = self
         collectionView.dataSource = self
-        collectionView.register(cellType: cell)
+        
+        switch type {
+        case .medias:
+            collectionView.register(PersonMediaCollectionViewCell.self, forCellWithReuseIdentifier: PersonMediaCollectionViewCell.reuseIdentifier)
```

Comment:
we have an extension for registering and dequeuing called `Reusable`:
```suggestion
                    collectionView.register(cellType:PersonMediaCollectionViewCell.self
```